ota-plus-web/app/reactapp-new/src/i18n.js

```diff
@@ -1,52 +1,21 @@
-import i18n from 'i18next';
-import Backend from 'i18next-xhr-backend';
+import i18n from "i18next";
+import { initReactI18next } from "react-i18next";
 import LanguageDetector from 'i18next-browser-languagedetector';
-import { initReactI18next } from 'react-i18next';
+import translations from '../../assets/locales/en/translation-new.json'
 
 i18n
-  // load translation using xhr -> see /public/locales
-  // learn more: https://github.com/i18next/i18next-xhr-backend
-  .use(Backend)
-  // detect user language
-  // learn more: https://github.com/i18next/i18next-browser-languageDetector
   .use(LanguageDetector)
-  // pass the i18n instance to react-i18next.
-  .use(initReactI18next)
-  // init i18next
-  // for all options read: https://www.i18next.com/overview/configuration-options
+  .use(initReactI18next) // passes i18n down to react-i18next
   .init({
-    // please see: https://www.i18next.com/overview/configuration-options
-    whitelist: ['en', 'zh'],
```

Comment:
Please do not remove language detection.

Reply:
What's the reason of whitelisting only 'en' and 'zh'?

Reply:
This was and is a functional requirement. When app will detect any other language it will fallback to en.